extensions/blocks/opentable/opentable.php

```diff
@@ -63,7 +62,7 @@ function set_availability() {
 			'missing_plan',
 			array(
 				'required_feature' => 'opentable',
-				'required_plan'    => 'premium-plan',
+				'required_plan'    => 'value_bundle',
```

Comment:
Should it use `REQUIRED_PLAN` here?

```suggestion
				'required_plan'    => REQUIRED_PLAN,
```